README.md

```diff
@@ -26,30 +28,31 @@ public static void SitOnAChair()
     
     AssignCommand(player, () => ActionSit(chair));
 }
-
 ```
-4. Configure your NWNX server with the following environment variables. Setting up an NWNX server is outside the scope of this guide - more information can be found in the Additional Resources section of this guide.
 
-```csharp
+- Configure your NWNX server with the following environment variables. Setting up an NWNX server is outside the scope of this guide - more information can be found in the Additional Resources section of this guide.
+
+```cs
 NWNX_DOTNET_ASSEMBLY=/nwn/home/dotnet/NWN.Framework.Lite 
 NWNX_DOTNET_ENTRYPOINT=NWN.Framework.Lite.Internal
 ```
 
-5. Add the script "sit" to the OnUsed event of a placeable chair.
-6. Build the project and run the NWNX server
-7. Log in and click the chair. Your character should sit down.
+- Add the script "sit" to the OnUsed event of a placeable chair.
```

Comment:
Let's keep the step numbers so we can easily reference them. Will help with troubleshooting if we can ask "What step are you on?"